packages/api/cms-api/src/access-log/access-log.interceptor.ts

```diff
@@ -1,8 +1,8 @@
 import { CallHandler, ExecutionContext, Inject, Injectable, Logger, NestInterceptor, Optional } from "@nestjs/common";
 import { GqlExecutionContext } from "@nestjs/graphql";
 import { GraphQLResolveInfo } from "graphql";
+import { CurrentUser } from "src/user-permissions/dto/current-user";
```

Comment:
Doesn't this cause problems during compilation?

Reply:
fe4b7eeb74ce3d71fd4bd73bb1566371cf7a644d